### PR TITLE
Bug 1163179 - [Calendar] modify and create account views should not depend on the stores r=gaye

### PR DESCRIPTION
**Pull request by millermedeiros**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5dc1967db470a1669beb829224bd58d50939ffac

File: apps/calendar/js/bridge.js [hunk 1]
```diff
@@ -52,16 +52,14 @@ exports.observeCalendars = function() {
...
   calendarStore.on('remove', getAllAndWrite);
   calendarStore.on('update', getAllAndWrite);
 
   stream.cancel = function() {
     calendarStore.off('add', getAllAndWrite);
-    calendarStore.off('preRemove', getAllAndWrite);
     calendarStore.off('remove', getAllAndWrite);
     calendarStore.off('update', getAllAndWrite);
-    stream._emitter.removeAllListeners();
+    stream._cancel();
```
Comment (gaye) [inline apps/calendar/js/bridge.js]: Still not sure about this thing?

File: apps/calendar/js/bridge.js [hunk 2]
```diff
@@ -91,10 +94,85 @@ var persistEvent = co.wrap(function *(event, action, capability) {
   return provider[action + 'Event'](event.data);
 });
 
-exports.getSetting = co.wrap(function *(id) {
-  var settingsStore = core.storeFactory.get('Setting');
-  return yield settingsStore.getValue(id);
-});
+exports.getSetting = function(id) {
```
Comment (gaye) [inline apps/calendar/js/bridge.js]: Hrm I think I already reviewed this part in another pr

Comment (gaye) [inline apps/calendar/js/bridge.js]: Will wait for you to land the other bit before reviewing

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b565ba6e422ce8b0e1df186bc4139a4bcafa7694

File: apps/calendar/js/bridge.js [hunk 3]
```diff
@@ -169,6 +179,11 @@ exports.observeAccounts = function() {
   return stream;
 };
 
+exports.availablePresets = function(presetList) {
```
Comment (gaye) [inline apps/calendar/js/bridge.js]: Maybe a comment about what this is since it's not obvious like the others?

Comment (millermedeiros) [inline apps/calendar/js/bridge.js]: Ok, reason for this is just because we can't create multiple local calendars.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d9b3b9b26413d0c19ae68c1ae5665fad22ed8c21

**Pull request merged**